crates/burn-tensor/src/tests/quantization/ops/remainder.rs

```diff
@@ -8,6 +8,43 @@ mod tests {
 
     #[test]
     fn should_support_remainder_basic() {
+        let device = Default::default();
+        let lhs = TestTensor::<1>::from_data(
+            // [-3.0, -2.0, -1.0, 1.0, 2.0, 3.0]
+            TensorData::quantized(
+                vec![-128i8, -85, -42, 42, 85, 127],
+                [6],
+                QuantizationStrategy::PerTensorAffineInt8(AffineQuantization::init(0.023529, 0)),
+            ),
+            &device,
+        );
+        let rhs = TestTensor::<1>::from_data(
+            // [2.0, 3.0, 1.0, 2.0, 1.0, 3.0]
+            TensorData::quantized(
+                vec![42i8, 127, -43, 42, -43, 127],
+                [6],
+                QuantizationStrategy::PerTensorAffineInt8(AffineQuantization::init(0.011749, -128)),
+            ),
+            &device,
+        );
+
+        let output = lhs.remainder(rhs);
+        let expected = TensorData::from([
+            0.983006477355957,
+            0.9960784912109375,
+            0.010457515716552734,
+            0.9882352948188782,
+            0.0026143789291381836,
+            2.9882352352142334,
+        ]);
```

Comment:
The expected value should be `[1., 1., -0., 1., 0., 0.]`, no? For `[-3.0, -2.0, -1.0, 1.0, 2.0, 3.0]` % `[2.0, 3.0, 1.0, 2.0, 1.0, 3.0]`.

The result should be close to the same operation in floating point representation, minus some precision errors due to de/quantization.

The first 5 values have de/quantization errors but why is the last one off? Is it a quantization error?

Reply:
It should be a quantization error because lhs and rhs have different value ranges. After dequantizing, the last element of lhs is 2.9882352, and 2.9960785 for rhs. The result is close to 3 (or 0 if modulo 3), so I think it's not that off.

Reply:
Ahhh ok thanks for the details! Makes sense in this case